### PR TITLE
fix(plugin-ledger-connectors): fixes 1445 and implementing correct interface types

### PR DESCRIPTION
**Pull request by m-courtin**
For all the connector class definitions the implementation are fixed to
follow their interface definition and to match in consistent way for all
the connectors

Closes: #1445
Signed-off-by: Michael Courtin <michael.courtin@accenture.com>

### CODE REVIEW

File: packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts [hunk 1]
```diff
@@ -108,7 +107,7 @@ export class PluginLedgerConnectorBesu
   implements
     IPluginLedgerConnector<
       DeployContractSolidityBytecodeV1Request,
-      DeployContractSolidityBytecodeV1Response,
+      RunTransactionResponse,
```
[the remark this answers is not in the document]
Comment (petermetz) [reply packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts]: @m-courtin What about the aliasing of the deploy req/res types to point to the run tx req/res types?

**Review by petermetz — COMMENTED** [review]
^^

File: packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (m-courtin) [reply packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts]: @petermetz There is no aliasing needed as the besu connector is using for the tx response and the deploy response the `transactionReceipt: Web3TransactionReceipt` in an object notation and the xdai connector is doing the same and using a `transactionReceipt: Web3TransactionReceipt`  in an object notation.

[the remark this answers is not in the document]
Comment (petermetz) [reply packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts]: @m-courtin Gotcha, thank you very much for looking into it!

**Review by petermetz — APPROVED** [review]
@m-courtin LGTM

**Pull request merged**